netstat/netstat_linux.go

```diff
@@ -253,8 +255,10 @@ func doNetstat(path string, fn AcceptFn) ([]SockTabEntry, error) {
 	if err != nil {
 		return nil, err
 	}
+	if f != nil {
+		defer f.Close()
+	}
```

Comment:
Here, again, the same situation as above. Nil-check is unnecessary here as the
os.Open already succeeded.